services/graphql-server/src/graphql/definitions/platform/content/types/company.js

```diff
@@ -22,10 +22,63 @@ type ContentCompany implements Content & PrimaryCategory & Contactable & Address
   salesChannels: String @projection
   productSummary: String @projection
 
+  youtube: YoutubeSettings @projection
+  youtubeVideos(input: YoutubeVideosInput = {}): YoutubePlaylistItemsApiResponse! @projection(needs: ["youtube"])
```

Comment:
The input should mimic the others: `ContentCompanyYoutubeVideosInput`

The return type should be specifc: `ContentCompanyYoutubeItems` - remove API - graphql doesn't care where the data came from.